ghost/core/core/server/data/migrations/versions/5.59/2023-08-07-11-17-05-add-posts-published-at-index.js

```diff
@@ -0,0 +1,50 @@
+const logging = require('@tryghost/logging');
+const {createNonTransactionalMigration} = require('../../utils');
+
+const INDEX_NAME = 'posts_published_at_index';
+
+module.exports = createNonTransactionalMigration(
+    async function up(knex) {
+        let hasIndex = false;
+
+        if (knex.client.config.client === 'sqlite3') {
```

Comment:
It'd be great if we could refactor to use `@tryghost/database-info` like https://github.com/TryGhost/Ghost/blob/3fa3a34187cdbad75f1831d743d4d855737d6b51/ghost/core/core/server/data/migrations/versions/4.0/29-fix-foreign-key-for-members-stripe-customers-subscriptions.js#L7